sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -317,7 +317,7 @@ func (m *networkFlowManager) Notify(e common.SensorComponentEvent) {
 	if !features.SensorCapturesIntermediateEvents.Enabled() {
 		log.Info(common.LogSensorComponentEvent(e))
 		switch e {
-		case common.SensorComponentEventCentralReachable:
+		case common.SensorComponentEventSyncFinished:
```

Comment:
Is it okay for `m.centralReady.Signal()` to wait for Sync finished?

I think, we may want to split it - `m.enricherTicker.Reset(tickerTime)` here and `m.centralReady.Signal()` into `SensorComponentEventCentralReachable`.

Wdyt?

Reply:
I don't think it's necessary. The `centralReady` signal is only used inside the ticker loop. So, if the loop is not running the signal won't be read.

I've changed a little bit the code though because this is only executed if we are not capturing intermediate events. If we are, the ticker was running from the start.

Reply:
Okay, so it is completely fine to not react on `SensorComponentEventCentralReachable` in this component?

Reply:
Yes. In fact we should check other components because waiting for the sync makes more sense in a lot of them.